apps/worker/src/app/workflow/services/standard.worker.ts

```diff
@@ -45,7 +41,7 @@ export class StandardWorker extends StandardWorkerService implements INovuWorker
 
     this.initWorker(this.getWorkerProcessor(), this.getWorkerOptions());
 
-    this.worker.on('failed', async (job: Job<IJobData, void, string>, error: Error): Promise<void> => {
+    this.worker.on('failed', async (job: Job<IStandardDataDto, void, string>, error: Error): Promise<void> => {
```

Comment:
we will need either merge IJobData and IStandardDataDto, or connect them, as they share the same interface.